docs/data/projects.yml

```diff
@@ -260,3 +260,29 @@
   gh: brentp/cyvcf2
   ci: [github]
   os: [apple, linux]
+
+- name: CTranslate2
+  gh: OpenNMT/CTranslate2
+  ci: [github]
+  os: [apple, linux]
+  notes: Includes libraries from the [Intel oneAPI toolkit](https://software.intel.com/content/www/us/en/develop/tools/oneapi/base-toolkit.html). The Linux wheels also include CUDA libraries for GPU execution.
+
+- name: MyPy
+  gh: mypyc/mypy_mypyc-wheels
+  stars: python/mypy
+  ci: [github]
+  os: [apple, linux, windows]
+  notes: MyPyC, the compiled component of MyPy.
+
+- name: Imagecodecs (fork)
+  gh: czaki/imagecodecs_build
+  stars: cgohlke/imagecodecs
+  ci: [github]
```

Comment:
Azure pipelines. GHA does not allow to restart a single job (sometimes failed because of network).

Reply:
Oops, thought I checked all these, thanks for the catch!